MARKETPLACE.md

```
@@ -0,0 +1,72 @@
+Discover and install plugins, adapters, and tools for NebulaDB.
+
+## 🚀 Official Packages
+
+### Core
+| Package | Version | Description |
+|---------|---------|-------------|
+| `@nebula-db/core` | [npm](https://www.npmjs.com/package/@nebula-db/core) | Core database engine |
+| `@nebula-db/nebula-db` | [npm](https://www.npmjs.com/package/@nebula-db/nebula-db) | All-in-one package |
+
+### Storage Adapters
+| Package | Version | Description |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)